beetsplug/convert.py

```diff
@@ -336,14 +338,31 @@ def copy_album_art(self, album, dest_dir, path_formats, pretend=False):
                            util.displayable_path(album.artpath))
             return
 
-        if pretend:
-            self._log.info(u'cp {0} {1}',
```

Comment:
We lost this nice shell-like output in pretend mode. Can we use this inside the `else` block for `resize` below when we're in pretend mode?